create-cluster.sh

```diff
@@ -68,6 +101,30 @@ getFIP() {
   echo "$_fip"
 }
 
+# validate a given flavor is available
+isFlavor() {
+  local _flavor=$1
+  if ! openstack flavor show ${_flavor} > /dev/null 2>&1; then
+    echo "ERROR: cluster flavor does not exist: ${_flavor}"
+    exit 2
+  fi
+}
+
+# validate if the given ip address range is already used
+isMachineNetUsed() {
+  local netIP=$1
+  local netName=$2
+
+  if openstack floating ip list -c "Fixed IP Address" -f value \
```

Comment:
This won't check all the CIDR ranges to make sure the network isn't in another range, but I think that's outside the scope of a bash script and this check will catch most cases.